data/presets/README.md

```diff
@@ -89,7 +89,7 @@ names when identifying the icon to be used for a given preset.
 To build presets, all you need to do is run `make`.
 
 This command will take care of running the build script, which packages all presets
-together with imagery data, and deprecated or discarded tags into one file, `data/data.js`,
+together with imagery data, and deprecated or discarded tags into one file, `dist/presets.js`,
```

Comment:
Good catch, the presets are in their own file now.  Can you also remove mention of the "imagery data, and deprecated or discarded tags"?